modules/cache/cache.go

```diff
@@ -34,25 +35,38 @@ func NewContext() error {
 		if conn, err = newCache(setting.CacheService.Cache); err != nil {
 			return err
 		}
-		const testKey = "__gitea_cache_test"
-		const testVal = "test-value"
-		if err = conn.Put(testKey, testVal, 10); err != nil {
+		err = Ping()
+		if err != nil {
```

Comment:
Would be better to use `if err = Ping(); err != nil {`

Reply:
No, ping doesn't work if the cache if full.

Reply:
I see the point, updated. `Ping` already does what it should do.